app/models/version.rb

```diff
@@ -204,6 +204,7 @@ def update_attributes_from_gem_specification!(spec)
       metadata: spec.metadata || {},
       requirements: spec.requirements,
       built_at: spec.date,
+      rubygems_version: (spec.required_rubygems_version || '').to_s,
```

Comment:
Ah I see. Well `nil.to_s == ""` so let's change this to match the line below. Better to keep similar style in this repo/method.